11ty/definitions/optical-allyship.md

```diff
@@ -0,0 +1,28 @@
+---
+title: optical allyship
+slug: optical-allyship
+defined: true
+speech: noun
+flag:
+  level: tool
+  text: White Supremacy Tool
+reading:
+  - text: 'We are not interested in Optical Allyship (origin)'
+    href: https://www.instagram.com/p/BiPDZkbFJFY/?hl=en
+  - text: 'If You Want To Be Anti-Racist, This Non-Optical Allyship Guide Is Required Reading'
+    href: https://www.vogue.co.uk/arts-and-lifestyle/article/non-optical-ally-guide
+  - text: 'What Is Optical Allyship? 3 Ways To Be Actively Anti-Racist'
+    href: https://www.bustle.com/p/what-is-optical-allyship-3-ways-to-be-actively-anti-racist-22956518
+  - text: 'What is optical allyship? Ways you can support the BLM movement apart from spreading hashtags'
+    href: https://www.msn.com/en-ca/news/politics/what-is-optical-allyship-ways-you-can-support-the-blm-movement-apart-from-spreading-hashtags/ar-BB15cJgz
+---
+
+> allyship that only serves at the surface level to platform the 'ally,' it makes a statement but doesn’t go beneath the surface and is not aimed at breaking away from the systems of power that oppress.
+
+## Related Words
+
+performative allyship, non-optical allyship
+
```

Comment:
It may benefit from adding an `## Examples` section to show a few examples of what this looks like?

Also, what about adding a comparison or relational section about the difference between performative allyship and optical allyship.

Reply:
Both good ideas. I'll need to study this a little deeper.